classlib/src/main/java/org/teavm/classlib/java/util/TBitSet.java

```diff
@@ -76,12 +86,19 @@ public static TBitSet valueOf(byte[] bytes) {
         return new TBitSet(ints);
     }
 
+    public static TBitSet valueOf(TByteBuffer buff) {
+        buff = buff.slice().order(TByteOrder.LITTLE_ENDIAN);
+        byte[] words = new byte[buff.remaining()];
+        buff.get(words);
+        return valueOf(words);
+    }
+
     public byte[] toByteArray() {
         byte[] bytes = new byte[(length + 7) / 8];
```

Comment:
I think there was an issue here. It should be `((length + 3) / 4)` (or better `((length + TInteger.SIZE - 1) / TInteger.SIZE)`

Reply:
Ah, no, it's ok